packages/zowe-explorer-api/src/fs/FsWatcher.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { Disposable as VSDisposable, FileSystemWatcher, Uri, workspace } from "vscode";
+
+interface CoreWatchers {
+    ds: FileSystemWatcher;
+    jobs: FileSystemWatcher;
+    uss: FileSystemWatcher;
+}
+
+export enum ZoweScheme {
+    DS = "zowe-ds",
+    Jobs = "zowe-jobs",
+    USS = "zowe-uss",
+}
+
+export class ZoweFsWatcher {
```

Comment:
I may be reviewing this a bit early or too late... but don't we want to add all of these to the Changelog?

Reply:
Thanks for catching this, addressed in 755ba5f0